CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,93 @@
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If a community member engages in unacceptable behavior, the community organizers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event).
+
+## 6. Reporting Guidelines
+
+If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer as soon as possible. speakherjp@gmail.com.
+
+
+Additionally, community organizers are available to help community members engage with local law enforcement or to otherwise help those experiencing unacceptable behavior feel safe.
+
+## 7. Addressing Grievances
+
+If you feel you have been falsely or unfairly accused of violating this Code of Conduct, you should notify WWCodeTokyo with a concise description of your grievance. Your grievance will be handled in accordance with our existing governing policies. [Policy](https://www.womenwhocode.com/codeofconduct)
+
+
+## 8. Scope
+
+We expect all community participants (contributors, paid or otherwise; sponsors; and other guests) to abide by this Code of Conduct in all community venues--online and in-person--as well as in all one-on-one communications pertaining to community business.
+
+This code of conduct and its related procedures also applies to unacceptable behavior occurring outside the scope of community activities when such behavior has the potential to adversely affect the safety and well-being of community members.
+
+## 9. Contact info
+
+speakherjp@gmail.com
+
+
+## 10. License and attribution
+
+The Citizen Code of Conduct is distributed by [Stumptown Syndicate](http://stumptownsyndicate.org) under a [Creative Commons Attribution-ShareAlike license](http://creativecommons.org/licenses/by-sa/3.0/). 
+
+Portions of text derived from the [Django Code of Conduct](https://www.djangoproject.com/conduct/) and the [Geek Feminism Anti-Harassment Policy](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy).
+
+_Revision 2.3. Posted 6 March 2017._
+
+_Revision 2.2. Posted 4 February 2016._
+
+_Revision 2.1. Posted 23 June 2014._
+
+_Revision 2.0, adopted by the [Stumptown Syndicate](http://stumptownsyndicate.org) board on 10 January 2013. Posted 17 March 2013._
+
+## 11. Addendum
+
+Any in-person events organized via Women Who Code, including events related to the SpeakHer project, are covered by the Women Who Code [Code of Conduct](https://www.womenwhocode.com/codeofconduct).
+
+Third-party events organized by users of the [SpeakHer Website](https://speakher.jp) may have their own community rules. In the scope of a Speaking Event where a speaker has been selected from the Speak Her DB, We request that the speaker is treated following the expectations in this Code of Conduct.
```

Comment:
I know we can't dictate their events, but I do expect that anyone who invites one of our speakers to their event treat that person with respect and dignity.